src/common/globalbroadcaster.hh

```diff
@@ -39,6 +39,7 @@ public:
   QString translateLineText{};
   //hold the dictionary id;
   QSet< QString > collapsedDicts;
+  //key hold the headword,  value holds the folders.
   QMap< QString, QSet< QString > > folderFavoritesMap;
```

Comment:
Maybe name this thing better?

https://www.google.com/search?q=map+naming+convention

- `wordFavoritesFoldersMap`
- `wordToFavoritesFolders`
- `FavoritesFoldersByWord`